packages/core/src/transports/utils.ts

```diff
@@ -1,9 +1,64 @@
+import type { EventEvent, ExceptionEvent, LogEvent, MeasurementEvent, TraceEvent } from '..';
+import { isArray } from '../utils';
+
 import { TransportItemType, transportItemTypeToBodyKey } from './const';
 import type { TransportBody, TransportItem } from './types';
 
-export function getTransportBody(item: TransportItem): TransportBody {
-  return {
-    [transportItemTypeToBodyKey[item.type]]: item.type === TransportItemType.TRACE ? item.payload : [item.payload],
-    meta: item.meta,
-  };
+export function getTransportBody(item: TransportItem | TransportItem[]): TransportBody {
```

Comment:
Is this new functionlity covered by a test somewhere? 
Otherwise I think it's good to add one.